packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -600,6 +604,24 @@ class FlutterValidator extends DoctorValidator {
       if (storageBaseUrl != null) {
         messages.add(ValidationMessage(_userMessages.flutterMirrorURL(storageBaseUrl)));
       }
+      // Add feature flags that are either enabled, or disabled compared to the default setting.
+      final List<String> featureFlags = <String>[
+        ..._featureFlags.allConfigurableFeatures
```

Comment:
Consider replacing splatting + list literal with a `.toList()` as that's shorter and less indented. I don't feel strongly about this, feel free to skip